packages/bot-web-ui/src/components/quick-strategy.jsx

```diff
@@ -35,7 +35,7 @@ const TradetypeOption = ({ type }) => (
     </div>
 );
 
-const popover_zindex = config.popover_zindex.quick_strategy;
+const popover_zindex = Number(config.popover_zindex.quick_strategy);
```

Comment:
Unnecessary typecast considering the numbers in constants are now actual numbers.

```suggestion
const popover_zindex = config.popover_zindex.quick_strategy;
```